common/network-common/src/main/java/org/apache/spark/network/server/TransportRequestHandler.java

```diff
@@ -67,6 +68,7 @@ public TransportRequestHandler(
       TransportClient reverseClient,
       RpcHandler rpcHandler,
       Long maxChunksBeingTransferred) {
+    super(reverseClient, rpcHandler.getStreamManager(), maxChunksBeingTransferred);
```

Comment:
do we still need the `maxChunksBeingTransferred` variable in this class?

Reply:
Yes, it is still in use in `processStreamRequest`.